pkg/config/runtime_config.go

```diff
@@ -20,6 +20,9 @@ const (
 	flagWatchNamespace          = "watch-namespace"
 	flagSyncPeriod              = "sync-period"
 	flagKubeconfig              = "kubeconfig"
+	flagCertDir                 = "cert-dir"
```

Comment:
lets use webhook-cert-dir, webhook-cert-file, webhook-key-file instead, that will make it clear that these configurations are intended for the webhook.